internal/codegen/golang/sqlite_type.go

```diff
@@ -56,6 +56,9 @@ func sqliteType(req *plugin.GenerateRequest, options *opts.Options, col *plugin.
 		}
 		return "sql.NullTime"
 
+	case "json", "jsonb":
+		return "[]byte"
```

Comment:
Is `json.RawMessage` preferred over `[]byte`?

Reply:
Yeah, it's what we do for PostgreSQL.

Reply:
Cool, that makes sense. Changed!